products/revenue_analytics/backend/hogql_queries/test/test_revenue_analytics_top_customers_query_runner.py

```diff
@@ -239,8 +239,8 @@ def test_with_data_group_by_all(self):
         self.assertEqual(
             results,
             [
-                ("John Doe", "cus_1", Decimal("313.7789"), "all"),
-                ("Jane Doe", "cus_2", Decimal("267.9514"), "all"),
+                ("John Doe", "cus_1", Decimal("239.9567749999"), "all"),
+                ("Jane Doe", "cus_2", Decimal("121.2037749998"), "all"),
```

Comment:
Money is spread across several months, and this is limiting months, so less revenue accrued in these months